hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/TestReconcileContainerCommandHandler.java

```diff
@@ -150,7 +154,17 @@ public void testReconcileContainerCommandMetrics(ContainerLayoutVersion layout)
       ReconcileContainerCommand cmd = new ReconcileContainerCommand(id, Collections.emptySet());
       subject.handle(cmd, ozoneContainer, context, null);
```

Comment:
I can't comment on the line directly but 152 has a comment from an older implementation we can remove here. I forgot to remove it in #7076 when we initially moved the timing handling to the supervisor